pldp/migrations/0009_auto_20190514_1450.py

```diff
@@ -0,0 +1,149 @@
+# Generated by Django 2.1.5 on 2019-05-14 14:50
+
+import django.contrib.gis.db.models.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pldp', '0008_auto_20190508_1706'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='location',
+            old_name='primary_name',
+            new_name='name_primary',
+        ),
+        migrations.RenameField(
+            model_name='location',
+            old_name='secondary_name',
+            new_name='name_secondary',
+        ),
+        migrations.RemoveField(
+            model_name='location',
+            name='area',
+        ),
+        migrations.RemoveField(
+            model_name='location',
+            name='line',
+        ),
+        migrations.RemoveField(
+            model_name='locationarea',
+            name='geometry',
+        ),
+        migrations.RemoveField(
+            model_name='locationline',
+            name='geometry',
+        ),
+        migrations.AddField(
+            model_name='location',
+            name='geometry',
+            field=django.contrib.gis.db.models.fields.GeometryField(default='POLYGON((-101.744384 39.32155, -101.552124 39.330048, -101.403808 39.330048, -101.332397 39.364032, -101.744384 39.32155))', help_text='Polygon or line that describes the geometry of the location. Polygons are intended for counts of people staying in an area while lines are intended forcounts of people moving across a threshold.', srid=4326),
```

Comment:
I wonder where this default came from? Not something we should necessarily change (the PLDP lists this as a required field so we'll need to have a default value anyway) but I'm curious.

Reply:
I copied this from a test fixture. Do you have a different SOP?

Reply:
My bad! I didn't know you added this default to the migration manually. Did you want to avoid hardcoding it as a `default` value on the `Location.geometry` field?

Reply:
Since it's a meaningless example it seemed safer to confine this to a migration instead of attaching it to the model. Here's the Django `makemigrations` help text: 
```
You are trying to change the nullable field 'geometry'to non-nullable without a default; we can't do that (the database needs something to populate existing rows). Please select a fix:
 1) Provide a one-off default now (will be set on all existing rows with a null value for this column)
 2) Quit, and let me add a default in models.py
```

This seems like more of a "one-off" situation!

Reply:
Gotcha, that makes sense to me, thanks! I've never actually used that option so I didn't know that it prepopulates the migration for you.